test/routes/Settings/routes/Account/components/test-account.js

```diff
@@ -6,177 +6,183 @@ import sinon from 'sinon';
-  });
-
-  it("Calls handlePassword when button is pressed", () => {
-    const handlePassword = sinon.spy();
-    const wrapper = shallow(<Account handlePassword={handlePassword} />);
-    const event = {
-      preventDefault: () => {}
-    };
-
-    wrapper.find({label: 'Change Password'}).simulate('click', event);
-    expect(handlePassword.calledOnce).to.equal(true);
-  });
-
-  it("Calls handleEmail when button is pressed", () => {
-    const handleEmail = sinon.spy();
-    const wrapper = shallow(<Account handleEmail={handleEmail} />);
-    const event = {
-      preventDefault: () => {}
-    };
+    it("Handles input changes", () => {
+      const wrapper = shallow(<Account />);
+      const event = {
+        target: {
+          value: 'test@ttu.edu',
+          name: 'email'
+        }
+      };
+
+      wrapper.instance().handleInputChange(event);
+      expect(wrapper.state().email).to.equal('test@ttu.edu');
+    });
 
-    wrapper.find({label: 'Change Email'}).simulate('click', event);
-    expect(handleEmail.calledOnce).to.equal(true);
-  });
+    it("Calls handlePassword when button is pressed", () => {
+      const handlePassword = sinon.spy();
+      const wrapper = shallow(<Account handlePassword={handlePassword} />);
+      const event = {
+        preventDefault: () => {}
+      };
 
-  it("Checks password onBlur, removes errors onFocus", () => {
-    const wrapper = shallow(<Account />);
-    const input = wrapper.find({name: 'password'});
-    const event = {
-      target: {
-        name: 'password'
-      }
-    };
-
-    wrapper.setState({
-      password: 'å'
+      wrapper.find({label: 'Change Password'}).simulate('click', event);
+      expect(handlePassword.calledOnce).to.equal(true);
     });
 
-    input.simulate('blur', event);
-    expect(wrapper.state().err.password).to.not.equal('');
+    it("Calls handleEmail when button is pressed", () => {
+      const handleEmail = sinon.spy();
+      const wrapper = shallow(<Account handleEmail={handleEmail} />);
+      const event = {
+        preventDefault: () => {}
+      };
 
-    input.simulate('focus', event);
-    expect(wrapper.state().err.password).to.equal('');
+      wrapper.find({label: 'Change Email'}).simulate('click', event);
+      expect(handleEmail.calledOnce).to.equal(true);
+    });
   });
 
-  it("Checks password confirmation onBlur, removes errors onFocus", () => {
-    const wrapper = shallow(<Account />);
-    const input = wrapper.find({name: 'confirm'});
-    const event = {
-      target: {
-        name: 'confirm'
-      }
-    };
-
-    wrapper.setState({
-      password: 'test',
-      confirm: 'asdf'
+  describe('Input checking', () => {
+    it("Checks password onBlur, removes errors onFocus", () => {
+      const wrapper = shallow(<Account />);
+      const input = wrapper.find({name: 'password'});
+      const event = {
+        target: {
+          name: 'password'
+        }
+      };
+
+      wrapper.setState({
+        password: 'å'
```

Comment:
I'm triggering the input validation function. It checks for non-ASCII characters